go/cmd/dolt/commands/diff.go

```diff
@@ -417,7 +448,7 @@ func fromNamer(name string) string {
 	return diff.From + "_" + name
 }
 
-func diffRows(ctx context.Context, newRows, oldRows types.Map, newSch, oldSch schema.Schema) errhand.VerboseError {
```

Comment:
I refactored this to hopefully make things a bit clearer.  This 150 line function grew out of control.  Even though I feel like this is better, this file needs another pass at some point (Probably the same as the rest of the codebase
).